source/development/client-api/cpp-client-programmers-guide.rst

```diff
@@ -1058,6 +1058,42 @@ alive.
         std::cout << "at least one dead/busy/locked/... device" << std::endl;
     }
 
+Enabling and disabling group members
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Devices belonging to a group can be temporarily excluded from all operations
+performed on the group using the *Group::disable* and *Group::enable* calls.
+
+Device name passed to the *disable* (*enable*) methods can contain
+wildcards (``*``).
+Note that only the first matching device will be disabled (enabled).
+
+The search algorithm is breadth-first search. All group elements are searched
+(in the insertion order) for a match before descending recursively to
+sub-groups. Recursive search can be disabled with a flag.
```

Comment:
If I understood well, this flag is the fwd optional boolean parameter (with default value = true) which can be provided to enable() and disable() methods.
Can we add this information somehow?

Reply:
I've added a reference to the "forwarding" section to this paragraph. Could you please check if it's ok now?